cluster/gce/util.sh

```diff
@@ -912,6 +912,16 @@ function detect-subnetworks() {
   echo "${color_red}Could not find subnetwork with region ${REGION}, network ${NETWORK}, and project ${NETWORK_PROJECT}"
 }
 
+function delete-all-firewall-rules-in-network() {
+  local -r all_firewalls=`gcloud compute firewall-rules list --project "${NETWORK_PROJECT}" --filter="network=${NETWORK}" --format="value(name)"`
+  if [[ -z "$all_firewalls" ]]; then
+    echo "No firewall rules left to be deleted or failed to list firewall rules."
```

Comment:
This is kind of bad, it either failed OR succeeded? Can we print an error message only when there was a problem?

Reply:
+1. Please see my suggestion in https://github.com/kubernetes/kubernetes/pull/53626/files#r143698905 which handles this.

Reply:
Fixed the logic.